katrain/gui/badukpan.py

```diff
@@ -227,7 +227,8 @@ def redraw(self, *_args):
         self.draw_board_contents()
 
     def draw_stone(
-        self, x, y, player, alpha=1, innercol=None, evalcol=None, evalscale=1.0, scale=1.0, ownership=None, loss=None
+            self, x, y, player, alpha=1, innercol=None, evalcol=None, evalscale=1.0, scale=1.0, ownership=None,
+            loss=None, depth=None
```

Comment:
use black -l 120

Reply:
what is this ,column size?

Reply:
If you mean the line width ,the function has exceed 120 .